doc/develop/west/alias.rst

```diff
@@ -0,0 +1,61 @@
+.. _west-aliases:
+
+West aliases
+############
+
+West allows to add alias commands to the local, global or system configuration files.
+These aliases make it easy to add shortcuts for frequently used, or hard to memorize
+commands for ease of development.
+
+Similar to how ``git`` aliases work, the alias command is replaced with the alias'
+full text and parsed as a new shell argument list (using the Python function
+`shlex.split()`_ internally to split the value). This enables adding argument
+parameters as they were passed to the original command. Spaces are considered
+argument separators, use proper escaping if arguments shouldn't be split.
+
+.. _shlex.split(): https://docs.python.org/3/library/shlex.html#shlex.split
+
+To add a new alias simply call the ``west config`` command:
+
+.. code-block:: console
+
+   west config alias.mylist "list -f '{name} {revision}'"
+
+To list aliases use ``west help [some_alias]``.
```

Comment:
Nit: the following gives a better/fancy output :)


```suggestion
To list aliases use :samp:`west help {some_alias}`.
```

https://www.sphinx-doc.org/en/master/usage/restructuredtext/roles.html#role-samp